portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -112,6 +116,77 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 
 				</ul>
 			</div>
+
+			<script data-senna-track="temporary" type="text/javascript">
```

Comment:
We should use `<aui:script sandbox="<%= true %>">` here, or wrap content of script in IIFE like [here](https://github.com/liferay/liferay-portal/blob/master/portal-web/docroot/html/taglib/ui/input_date/page.jsp#L178-L179). Without this, we are polluting global namespace with all declared variables.

Reply:
Thanks @markocikos